### PR TITLE
New attribute type: plain text

### PR DESCRIPTION
**Pull request by IKarbowiak**
Add plain text attribute.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6760 Add plain text attribute](https://app.clickup.com/t/2549495/SALEOR-6760)

**Review by fowczarek — APPROVED** [review]

**Review by SzymJ — APPROVED** [review]

File: saleor/graphql/attribute/mutations.py [hunk 1]
```diff
@@ -192,6 +192,7 @@ def prepare_operations(cls, moves: ReorderInput, values: "QuerySet"):
 class AttributeValueInput(graphene.InputObjectType):
     value = graphene.String(description=AttributeValueDescriptions.VALUE)
     rich_text = JSONString(description=AttributeValueDescriptions.RICH_TEXT)
+    plaint_text = graphene.String(description=AttributeValueDescriptions.PLAIN_TEXT)
```
Comment (maarcingebala) [inline saleor/graphql/attribute/mutations.py:195]: Typo: `plaint_text`

File: saleor/graphql/attribute/types.py [hunk 2]
```diff
@@ -387,6 +390,7 @@ class AttributeValueInput(graphene.InputObjectType):
         required=False,
     )
     rich_text = JSONString(required=False, description="Text content in JSON format.")
+    plain_text = graphene.String(required=False, description="Text content.")
```
Comment (maarcingebala) [inline saleor/graphql/attribute/types.py:393]:
```suggestion
    plain_text = graphene.String(required=False, description="Plain text content.")
```

File: saleor/graphql/translations/types.py [hunk 3]
```diff
@@ -69,6 +69,7 @@ class AttributeValueTranslation(BaseTranslationType):
     id = graphene.GlobalID(required=True)
     name = graphene.String(required=True)
     rich_text = JSONString(description="Attribute value." + RICH_CONTENT)
+    plain_text = graphene.String(description="Attribute text value.")
```
Comment (maarcingebala) [inline saleor/graphql/translations/types.py:72]:
```suggestion
    plain_text = graphene.String(description="Attribute plain text value.")
```

File: saleor/graphql/translations/types.py [hunk 4]
```diff
@@ -79,6 +80,7 @@ class AttributeValueTranslatableContent(ModelObjectType):
...
```
Comment (maarcingebala) [inline saleor/graphql/translations/types.py:83]:
```suggestion
    plain_text = graphene.String(description="Attribute plain text value.")
```

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**